water/main.py

```diff
@@ -134,6 +134,23 @@ def update_node_highlight(attrname, old, new):
             # Remove history plot from layout for the graph and widgets
             layout.children[0] = layout_row
 
+    def update_click_node(event):
+        nodes_clicked_ints = graph.node_renderer.data_source.selected.indices
+        # It's possible to click multiple nodes when they overlap,
+        # but we only want one
+        first_clicked_node_int = nodes_clicked_ints[0]
+        clicked_node = "None"
+        node_count = -1
+        for node in G.nodes():
+            node_count += 1
+            if node_count == first_clicked_node_int:
+                clicked_node = node
+                break
+        if what_click_does.active == 0:
+            pollution_history_select.value = clicked_node
+        if what_click_does.active == 1:
+            pollution_location_select.value = clicked_node
```

Comment:
Can we have move informative labels that 0 and 1.

It looks like these must be integers, but something like
```python
POLLUTION_HISTORY = 0
POLLUTION_LOCATION = 1

...

what_click_does.active == POLLUTION_HISTORY:
...
```
would be more readable.